web/src/assets/locales/en.json

```diff
@@ -82,29 +82,29 @@
         },
         "allow_pr": {
           "allow": "Allow Pull Requests",
-          "desc": "Pipelines can run on pull requests."
+          "desc": "Allows pipelines to run on pull request events."
         },
         "allow_deploy": {
           "allow": "Allow deployments",
-          "desc": "Allow deployments from successful pipelines. Only use if you trust all users with push access."
+          "desc": "Allow deployments from successful pipelines. Use this option only if you trust all users with push permissions."
         },
         "netrc_only_trusted": {
-          "netrc_only_trusted": "Only inject netrc credentials into trusted clone plugins",
-          "desc": "If enabled, git netrc credentials are only available for trusted clone plugins set in `WOODPECKER_PLUGINS_TRUSTED_CLONE`. Otherwise, all clone plugins can use the netrc credentials. This option has no effect on non-clone steps."
+          "netrc_only_trusted": "Only inject git credentials into trusted clone plugins",
+          "desc": "When enabled, Git credentials are accessible only to trusted clone plugins specified in WOODPECKER_PLUGINS_TRUSTED_CLONE. Otherwise, custom clone plugins can use git credentials. This setting has no affect on non-clone steps."
```

Comment:
```suggestion
          "desc": "When enabled, git credentials are accessible only to trusted clone plugins specified in WOODPECKER_PLUGINS_TRUSTED_CLONE. Otherwise, custom clone plugins can use git credentials. This setting has no affect on non-clone steps."
```